.goreleaser.yml

```diff
@@ -0,0 +1,19 @@
+build:
+  binary: terraform-provider-nexus_v{{ .Version }}
+  goos:
+  - darwin
+  - linux
+  - windows
```

Comment:
Is windows supported by makefile? (see https://github.com/datadrivers/terraform-provider-nexus/blob/master/GNUmakefile#L3)

Reply:
`goreleaser` does not use the GNUmakefile. It builds using standard `go` build commands.